src/Repository/TagRepository.php

```diff
@@ -85,6 +88,36 @@ public function findAllFlatTagsWithNbEntries($userId)
             ->getArrayResult();
     }
 
+    /**
+     * Find all tags per user with nb entries.
+     *
+     * @param int $userId
+     *
+     * @return array<array{tag: Tag, nbEntries: int}>
+     */
+    public function findAllTagsWithNbEntries($userId)
```

Comment:
I absolutely wanted that the TagVoter votes on real entity instances.
But the list of tags used `findAllFlatTagsWithNbEntries` that returns an array, making the is granted calls for edit and delete attributes fail.
so introduced this that returns a real entity instance, in addition to the number of entries.

Side note: on fact the tag entity is not used at all in the voter, as for now the grant is given only because being logged in. But in the futur this will certainly change when tags will be owned by the user, see #6784